### PR TITLE
Adds Vagrantfile for Eclipse itegration

### PR DESCRIPTION
**Pull request by navidshaikh**
- Adds Vagrantfile for Eclipse integration for forwarding guest port 2376 to host port 2379
- This Vagrantfile can be shipped as part of zip file

### CODE REVIEW

**Comment by navidshaikh** [comment]
@whitel @LalatenduMohanty : Does this make sense? 

Having a ready populated Vagrantfile with help ease the Eclipse integration.

**Comment by LalatenduMohanty** [comment]
@navidshaikh LGTM!

**Comment by langdon** [comment]
Shouldn't the Vagrantfile test that the adb-info plugin is installed? (e.g. http://stackoverflow.com/questions/21001567/how-to-detect-from-a-vagrantfile-if-a-plugin-has-been-installed)

File: components/centos/centos-docker-eclipse/Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+#Vagrant file for libvirt/kvm and virtualbox provider
+
+# check if plugin is installed on system
+unless Vagrant.has_plugin?("vagrant-adbinfo")
+  raise "vagrant-adbinfo plugin is not installed, run `vagrant plugin install vagrant-adbinfo` to install the plugin."
+end
```
Comment (navidshaikh) [inline components/centos/centos-docker-eclipse/Vagrantfile:9]: @whitel : Added the required changes.

**Comment by langdon** [comment]
LGTM

**Pull request merged**